packages/xfsprogs/xfsprogs.spec

```
@@ -0,0 +1,91 @@
+Name: %{_cross_os}xfsprogs
+Version: 6.3.0
+Release: 1%{?dist}
+Summary: Utilities for managing the XFS filesystem
+License: GPL-2.0-only and LGPL-2.1-only
+URL: https://xfs.wiki.kernel.org
+Source0: http://kernel.org/pub/linux/utils/fs/xfs/xfsprogs/xfsprogs-%{version}.tar.xz
+Patch1: 0001-libxfs-do-not-try-to-run-the-crc32selftest.patch
+
+BuildRequires: %{_cross_os}glibc-devel
+BuildRequires: %{_cross_os}libuuid-devel
+BuildRequires: %{_cross_os}libinih-devel
+BuildRequires: %{_cross_os}liburcu-devel
+BuildRequires: %{_cross_os}libblkid-devel
+
+Requires: %{_cross_os}liburcu
+Requires: %{_cross_os}libinih
+
+%description
+%{summary}.
+
+%package devel
+Summary: XFS filesystem-specific headers
+Requires: %{name}
+
+%description devel
+%{summary}.
+
+%prep
+%autosetup -n xfsprogs-%{version} -p1
+
+%build
+%cross_configure \
+  --enable-blkid=yes \
+  --enable-lto=no \
+  --enable-editline=no \
+  --enable-scrub=no
+
+%make_build
+
+%install
+make DIST_ROOT=%{buildroot} install install-dev \
+  PKG_ROOT_SBIN_DIR=%{_cross_sbindir} PKG_ROOT_LIB_DIR=%{_cross_libdir}
+
+rm -f %{buildroot}/%{_cross_libdir}/*.{la,a}
+
+%files
+%license LICENSES/GPL-2.0 LICENSES/LGPL-2.1
+%{_cross_attribution_file}
+%{_cross_libdir}/*.so.*
+%{_cross_sbindir}/fsck.xfs
+%{_cross_sbindir}/mkfs.xfs
+%{_cross_sbindir}/xfs_admin
+%{_cross_sbindir}/xfs_bmap
+%{_cross_sbindir}/xfs_copy
+%{_cross_sbindir}/xfs_db
+%{_cross_sbindir}/xfs_estimate
+%{_cross_sbindir}/xfs_freeze
+%{_cross_sbindir}/xfs_fsr
+%{_cross_sbindir}/xfs_growfs
+%{_cross_sbindir}/xfs_info
+%{_cross_sbindir}/xfs_io
+%{_cross_sbindir}/xfs_logprint
+%{_cross_sbindir}/xfs_mdrestore
+%{_cross_sbindir}/xfs_metadump
+%{_cross_sbindir}/xfs_mkfile
+%{_cross_sbindir}/xfs_ncheck
+%{_cross_sbindir}/xfs_quota
+%{_cross_sbindir}/xfs_repair
+%{_cross_sbindir}/xfs_rtcp
+%{_cross_sbindir}/xfs_spaceman
+%{_cross_datadir}/xfsprogs/mkfs/*.conf
+%exclude %{_cross_mandir}
+%exclude %{_cross_datadir}/doc/*
+%exclude %{_cross_datadir}/locale/*
+
+%files devel
+%dir %{_cross_includedir}/xfs
+%{_cross_includedir}/xfs/handle.h
+%{_cross_includedir}/xfs/jdm.h
+%{_cross_includedir}/xfs/linux.h
+%{_cross_includedir}/xfs/xfs.h
+%{_cross_includedir}/xfs/xfs_arch.h
+%{_cross_includedir}/xfs/xfs_fs.h
+%{_cross_includedir}/xfs/xfs_fs_compat.h
+%{_cross_includedir}/xfs/xfs_types.h
+%{_cross_includedir}/xfs/xfs_format.h
+%{_cross_includedir}/xfs/xfs_da_format.h
+%{_cross_includedir}/xfs/xfs_log_format.h
+%{_cross_includedir}/xfs/xqm.h
```

nit: this is fine too --
```suggestion
%dir %{_cross_includedir}/xfs
%{_cross_includedir}/xfs/*.h
```